examples/basic-form/src/components/TextBox/primitive.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-primitives';
+import styles from './style';
+import StrengthMeter from '../StrengthMeter';
+
+const TextBox = ({ label, value, type }: Props) => (
+  <View style={styles.formElement}>
+    <Text style={styles.label}>{label}</Text>
+    <View style={styles.textbox}>{value}</View>
+    {type === "password" && value.length > 0 && 
```

Comment:
You could also render `StrengthMeter` directly in your form - it doesn't need to live inside TextBox explicitly —

~=
```js
<View>
  <TextBox />
  <TextBox />
  <StrengthMeter />
  <Button />
</View>
```

Alternatively, if you wanted it to be attached to the TextBox,
```js
<View>
  <TextBox />
  <TextBox>
    <StrengthMeter />
  </TextBox>
  <Button />
</View>
```

& then in TextBox,
```
{ children && props.children }
```